projects/canopy/src/lib/forms/date/date-field.component.scss

```diff
@@ -13,6 +13,12 @@
   display: flex;
   flex-direction: column;
   margin-right: var(--space-sm);
+
+  & .lg-input-field {
```

Comment:
I think this selector could be all on one line: `& .lg-input-field .lg-input-field__inputs` since nothing else is using the first bit.

Reply:
Sorry, habit.  I'll update it now.  Also I've added evidence to the description